app/javascript/packages/document-capture/components/acuant-capture.tsx

```diff
@@ -135,18 +131,8 @@ export const isAcuantCameraAccessFailure = (error: AcuantCaptureFailureError): e
  * Returns a human-readable document label corresponding to the given document type constant.
  *
  */
-function getDocumentTypeLabel(documentType: AcuantDocumentType): AcuantDocumentTypeLabel | string {
-  switch (documentType) {
-    case 0:
-      return 'none';
-    case 1:
-      return 'id';
-    case 2:
-      return 'passport';
-    default:
-      return `An error in document type returned: ${documentType}`;
-  }
-}
+const getDocumentTypeLabel = (documentType: AcuantDocumentType): AcuantDocumentTypeLabel | string =>
+  AcuantDocumentType[documentType]?.toLowerCase() ?? `An error in document type returned: ${documentType}`;
```

Comment:
I'm guessing you already saw this in the build, but just a heads-up, there is a lint error in this file.

Reply:
whoops, fixed in 516b3c63f5031e23527ea714f2387c8a0cad3a9b